javascript/webpacker_react-npm-module/package.json

```diff
@@ -9,6 +9,7 @@
     "react": ">= 0.14",
     "react-dom": ">= 0.14"
   },
+  "files": ["webpack-config-for-hmr.js", "hmr.js"],
```

Comment:
Those files will need to move to `src/` to be compiled with Babel, I need to change the build step to handle this correctly (at the moment you need to use `import ... from 'webpacker-react/dist/hmr` which is not needed)